master/usage/configuration/bgp.md

```diff
@@ -144,27 +180,34 @@ spec:
 EOF
 ```
 
-To remove a global BGP peer that was configured with IP address 192.20.30.40
-(the AS number is not required when deleting) run the following command on any
-node:
+To view the current list of global peers run the following command.
```

Comment:
so this is not specific to global or node peers, you can just say `BGP peers` in this section since we treat them similarly in v3, so when you do a list it lists both global and node peers together, and updating and deleting process is the same for both, so no need to say `global` in this section